llmfoundry/eval/datasets/in_context_learning_evaluation.py

```diff
@@ -1292,25 +1474,13 @@ def tokenize_example(
 def build_icl_dataloader(
     icl_task_type: str,
     dataset_uri: str,
-    tokenizer: transformers.PreTrainedTokenizerBase,
+    tokenizer: Union[transformers.PreTrainedTokenizer,
```

Comment:
Any particular reason you changed the type here? Both of those classes inherit from the base class, so I've used the base type for tokenizer typing.

Reply:
Probably an artifact of some merge error by me, will revert it back